env.example

```diff
@@ -1,8 +1,15 @@
-export LOCATION=eastus
+export LOCATION=westeurope
 export ARO_IMAGE=arointsvc.azurecr.io/aro:latest
 export NO_CACHE=false
 export AZURE_EXTENSION_DEV_SOURCES="$(pwd)/python"
 
 export ARO_SKIP_PKI_TESTS=true
 
+export OPENSHIFT_VERSION="4.13.23"
+export CLUSTER_RESOURCEGROUP="v4-$LOCATION"
```

Comment:
nit: it likely doesn't matter since all this is local RP anyways - but it may be best to make the CRG also user-distinct, like `${USER}-rg`